app/src/main/java/com/kickstarter/features/search/ui/SearchAndFilterActivity.kt

```diff
@@ -60,7 +60,7 @@ class SearchAndFilterActivity : ComponentActivity() {
             filterMenuViewModelFactory = FilterMenuViewModel.Factory(env)
             filterMenuViewModel.getRootCategories()
 
-            val phase2ff = env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_SEARCH_FILTER) ?: false
+            val phaseff = env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_SEARCH_FILTER) ?: false
```

Comment:
Phase gating has changed, as phase 2 was successfully released, moving it now to gate phase 4.